controllers/appwrapper_controller.go

```diff
@@ -50,10 +50,7 @@ var (
 	useMachineSets   bool
 
 	maxScaleNodesAllowed int
-	msInformerHasSynced  bool
-	machineClient        mapiclientset.Interface
-	kubeClient           *kubernetes.Clientset
-	appwrapper           arbv1.AppWrapper
+	finalizerName        = "machineset.finalizers.example.com"
```

Comment:
Can be a `const`, also the convention for naming finaliser is to scope them by API group, like `mcad.codeflare.dev/name`.